virus/Puccinia_triticum.md

```diff
@@ -0,0 +1,109 @@
+# {Q1A Paper Title}
+
+written by: [Ryan Fu](https://github.com/yourname%20*optional%20link*)
+
+### {Puccinia triticum}
+
+(Puccinia for the fungus it resides in, triticum for the wheat plant the fungus infects)
+
+## Abstract
+
+Triticum aesitivum, or common wheat, as been a global food staple for centuries. The global reliance on the plant has resulted in wheat stripe, a disease that causes yellow stripes and discolorings, posing a massive threat to global food security. Currently, it is well known that fungal pathogens in the Puccinia species, namely Puccinia striiformis and Puccinia triticina, are primarily responsible for the infection. However, the advancement of sequencing technology seems to have unveiled a new mechanism for which fungal infeciton takes place. In this report, we characterize a novel virus - Puccinia triticum (PtV) that seems to be heavily correlated with wheat rust disease. Using serratus and NCBI databases, we were able to further sequence our viral genome and annotate it and uncovered a narnavirus with an ambigrammatic genome. Through this database, we also uncovered strong correlations with PsV5, an already discovred virus that seems to be a key contributor in wheat stripe rust. Taking it all together, our analysis provides a solid framework for uncovering the mechanism behind this phenomenom. Understanding how our novel virus functions and its structure can lead to developments in anti-viral treatments to combat wheat stripe rust and provide promote food security globally.
+
+## Results
+
+### Ecology subsection A...
+
+Analyzing the SRA runs obtained from serratus, containing our sequence, we can determine which species’ our virus most likely infects.
+
+![](img/PtV_images/Figure_1.0.png)
```

Comment:
For all of your figures, add a short figure description in the markdown so that the Display text is shown if the image is not rendered.